tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "experimentalDecorators": true,
```

Comment:
not sure if there was specific reason why it wasn't turned on before, WebStorm complained about decorators

Reply:
Nope, seems good.